Makefile

```diff
@@ -122,28 +122,12 @@ build-image:
 run: build-image
 	@CID=$(shell docker run --net=host -d --name $(SERVER) $(IMAGE)) && echo $${CID}
 
-test: test-storage test-influx test-stat test-logs test-project test-service test-stack
-
-test-storage:
-	@go test -v $(REPO)/data/storage/etcd
-
-test-influx:
-	@go test -v $(REPO)/data/influx
-
-test-stat:
-#	@go test -v $(REPO)/api/rpc/stat
-
-test-logs:
+test:
+#	@go test -v $(REPO)/api/rpc/build
 	@go test -v $(REPO)/api/rpc/logs
-
-test-project:
-	@go test -v $(REPO)/api/rpc/project
-
-test-service:
+#	@go test -v $(REPO)/api/rpc/project
```

Comment:
Let's get rid of `-v` and make tests silent. If tests are failing and people want to debug them, they can run the specific test manually with the option (ex: `go test github.com/appcelerator/amp/api/rpc/logs`).